specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.nodejs.md

```diff
@@ -0,0 +1,14 @@
+## Node.js
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  package-name: azure-arm-containerservicefleet
```

Comment:
New `readme.nodejs` for fleet, is this file actually used? Modified `package-name` and `output-folder`.